Spanner/src/Transaction.php

```diff
@@ -529,6 +545,9 @@ public function rollback(array $options = [])
      *
      *     @type array $mutations An array of mutations to commit. May be used
      *           instead of or in addition to enqueing mutations separately.
+     *     @type bool $returnCommitStats If true, commit statistics will be
+     *           returned and accessible via `commitStats`.
```

Comment:
```suggestion
     *           returned and accessible via {@see Google\Cloud\Spanner\Transaction::getCommitStats()}.
```

You might need to break the line up if that takes it over 120 chars.